testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/SpotBugsPluginSmokeTest.groovy

```diff
@@ -18,10 +18,13 @@ package org.gradle.smoketests
 
 import org.gradle.test.precondition.Requires
 import org.gradle.test.preconditions.UnitTestPreconditions
+import spock.lang.Ignore
 import spock.lang.Issue
 
 class SpotBugsPluginSmokeTest extends AbstractPluginValidatingSmokeTest {
 
+    // TODO: fix/remove in 9.0
+    @Ignore("Not compatible with breaking changes in Report class, see https://github.com/spotbugs/spotbugs-gradle-plugin/blob/master/src/main/kotlin/com/github/spotbugs/snom/SpotBugsReport.kt")
```

Comment:
Note: this plugin extends the Report class and overrides `getDisplayName`, I don't think we can workaround this.

Reply:
Good catch!
I am not sure how they can make their plugin work for Gradle < 9 and Gradle 9+ 🤔